samples/apps/copilot-chat-app/SKWebApi/appsettings.json

```diff
@@ -1,48 +1,42 @@
 {
-  // Consider populating secrets, such as "Key" properties, from dotnet's user secrets when running locally.
-  // https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-7.0&tabs=windows#secret-manager
-  // Ex: dotnet user-secrets set "CompletionConfig:Key" "MY_SECRET_VALUE"
-  // Values in user secrets take precedence over those in this file
+  /* Consider populating secrets, such as "Key" properties, from dotnet's user secrets when running locally.
+     https://learn.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-7.0&tabs=windows#secret-manager       
+     Values in user secrets take precedence over those in this file. */
 
   "ServicePort": "40443",
+
   "CompletionConfig": {
     "Label": "Completion",
-    "AIService": "[Enter 'AzureOpenAI', or 'OpenAI']",
-    "DeploymentOrModelId": "[Enter the Azure OpenAI deployment name or OpenAI model, e.g. 'text-davinci-003']",
-    "Endpoint": "[Enter the endpoint of your Azure OpenAI instance, e.g. https://contoso.openai.azure.com]"
-    /* Use user secrets or Key Vault - Putting secrets in a plain-text file
-       is a security risk and should be done as a last resort only. */
-    //"Key": "[DO NOT CHECK IN - Key to your (Azure) OpenAI instance.]"
+    "AIService": "AzureOpenAI", // Or "OpenAI" when using OpenAI directly
+    "DeploymentOrModelId": "text-davinci-003", // Azure OpenAI deployment name or OpenAI model to use
+    "Endpoint": "" // Your Azure OpenAI endpoint (e.g. https://contoso.openai.azure.com) - Leave blank when using OpenAI directly
+    /* "Key": Key for your (Azure) OpenAI instance. Use user secrets, environment variable or Key Vault to populate.
+       Putting secrets in a plain-text file is a security risk and should be done as a last resort only.
+       Ex: dotnet user-secrets set "CompletionConfig:Key" "MY_COMPLETION_KEY" */
   },
+
   // Remove or comment out EmbeddingConfig if no embedding backend is required
```

Comment:
Please add a line stating when/why no embedding backend would be required.

Reply:
Done